### DISCUSSION TITLE
solidity developer

### DISCUSSION THREAD

**Discussion by omerhayyam0202** [community/community#33257] [Other Feature Feedback, Questions, & Ideas] [answered]
hello,i am getting a code developer consultant.
those who are interested. [redacted] They can reach with.I think we can agree

**Comment by queenofcorgis** [comment, marked as answer]
Hi @omerhayyam0202, we're happy you're here 😀

However, we currently do not allow job listings on the Community on Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!

Thanks for your interest 👍
